python/amici/petab_objective.py

```diff
@@ -63,6 +65,8 @@ def simulate_petab(
     :param simulation_conditions:
```

Comment:
add: meaningless if edatas and parameter_mapping is passed

Reply:
added comment, and generate below only if required.